source/exercises/catalog-analysis.md

```diff
@@ -0,0 +1,124 @@
+---
+jupytext:
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.13.0
+kernelspec:
+  display_name: Python 3
+  language: python
+  name: python3
+---
+
+# 分析地震目录
+
+- 本节贡献者: {{田冬冬}}（作者）、{{姚家园}}（审稿）
+- 最近更新日期: 2022-07-31
+- 预计花费时间: 15 分钟
+
+---
+
+在获取地震目录之后，通常还需要对地震目录做一些简单的绘图和分析。这一节演示如何
+绘制最简单的地震深度直方图、地震震级直方图以及震级-频次关系图。
+
+首先，需要把地震目录准备好。这里我们选择下载 2000 - 2010 年间震级大于 4.8 级，
+震源深度大于 50 km 的地震。这一地震目录中共计约 7000 个地震：
+```{code-cell} ipython3
+from obspy.clients.fdsn import Client
+
+client = Client("USGS")
```

Comment:
See https://seismo-learn.org/sitepreview/seismo-learn/seismology101/obspy-get-events/exercises/catalog-analysis/ for an bug.

Reply:
It's due to temporary network issues so can be safely ignored.